src/commands/step-functions/helpers.ts

```diff
@@ -0,0 +1,52 @@
+import {StepFunctions} from 'aws-sdk'
+
+export const buildArn = (
+  partition: string,
+  service: string,
+  region: string,
+  accountId: string,
+  resourceType: string,
+  resourceId: string
+): string => {
+  return `arn:${partition || 'aws'}:${service}:${region}:${accountId}:${resourceType}:${resourceId}`
+}
+
+export const buildLogGroupName = (stepFunctionName: string, env: string | undefined): string => {
+  return `/aws/vendedlogs/states/${stepFunctionName}-Logs${env !== undefined ? '-' + env : ''}`
+}
+
+export const buildSubscriptionFilterName = (stepFunctionName: string): string => {
+  return `${stepFunctionName}LogGroupSubscription`
+}
+
+export const isValidArn = (str: string | undefined): boolean => {
+  return typeof str === 'string' && str.indexOf('arn:') === 0 && str.split(':').length >= 6
+}
```

Comment:
Could we switch the logic to be more readable?
Also, when is this method going to receive an `undefined`?

Reply:
I removed the `undefined` type. Is the change below what you were thinking for improved readability?

```typescript
export const isValidArn = (str: string): boolean => {
  const hasArnPrefix = str.indexOf('arn:') === 0
  const hasAtLeastSixFields = str.split(':').length >= 6

  return hasArnPrefix && hasAtLeastSixFields
}
```